tools/resourcedocsgen/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/pulumi/docs/tools/resourcedocsgen
```

Comment:
@EvanBoyle this is what I added. Moreover, trying to run this tool from any other path other than this folder always seems to fail. So I had to adjust the `gen_resource_docs.sh` script accordingly.